content/docs/1.8.0/advanced-resources/system-backup-restore/backup-longhorn-system.md

```diff
@@ -38,6 +38,8 @@ It includes below resources associating with the Longhorn system:
 - StorageClasses
 - Volumes
 
+> **Note:** The default backup target (`default`) is always used to store system backups.
+
 > **Note:** Longhorn is unable to back up V2 Data Engine backing images.
 
 > **Note:** Longhorn does not backup `Nodes`. The Longhorn manager on the target cluster is responsible for creating its own Longhorn `Node` custom resources.
```

Comment:
Too many `Note`. Please use one not for the four items

![image](https://github.com/user-attachments/assets/a962b451-ad91-4d0b-b6cd-6f2171839105)


> Note
>
> - ...
>
> - ....

Reply:
Modified. Thanks.